main.tf

```diff
@@ -57,6 +57,23 @@ resource "null_resource" "icp-cluster" {
 
 ## icp-boot-preconfig hooks are run before icp-docker, if defined
 
+# To make script parameters more consistent we'll define a common set here
```

Comment:
I quite like this approach for general readability, assuming it doesn't have any ill effects on scripts getting loads of options that's not of use for them.

Reply:
I do not think that there will be any ill effects from the number of arguments passed, but the argument parsing might run out of single-letter parameter options.

```
$ getconf ARG_MAX
262144
```